packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.spec.js

```diff
@@ -26,21 +34,51 @@ describe('Tasks', function () {
   })
 
   it('should render null if the workflow is load but has no tasks', function () {
-    const wrapper = shallow(<Tasks.wrappedComponent loadingState={asyncStates.success} />)
+    const wrapper = shallow(<Tasks.wrappedComponent loadingState={asyncStates.success} ready={true} />)
     expect(wrapper.type()).to.be.null
   })
 
   it('should render the correct task component if the workflow is loaded', function () {
-    const tasks = [{
-      answers: [{ label: 'yes' }, { label: 'no' }],
-      question: 'Is there a cat?',
-      required: true,
-      taskKey: 'init',
-      type: 'single'
-    }]
-
-    const wrapper = shallow(<Tasks.wrappedComponent loadingState={asyncStates.success} tasks={tasks} />)
+    const wrapper = shallow(<Tasks.wrappedComponent loadingState={asyncStates.success} ready={true} tasks={tasks} />)
     // Is there a better way to do this?
     expect(wrapper.find('inject-SingleChoiceTask')).to.have.lengthOf(1)
   })
+
+  describe('when the subject is loading', function () {
```

Comment:
Maybe this should be describing a thing, rather than a state. Then the `it` block can test an expected state for that thing.